tasks/CarryMyLuggage.tex

```diff
@@ -59,8 +59,10 @@ \subsection*{Additional Rules and Remarks}
     \begin{enumerate}
      \item Natural interaction (e.g., waving and calling)
      \item Unnatural interaction (e.g., raising both hands and jumping)
+	 \item Coming back (e.g., asking the operator to step back in front of the robot)
      \item Touching the robot (e.g., pulling the robot's hand)
     \end{enumerate}
+	Note: Penalties for human assistance are only applied if the robot continues to score after the interaction.
```

Comment:
Can the penalties be applied multiple times? How do different DEM penalties stack - (e.g. up to XXXpts penalty) We should specify that.

Id suggest something like " Penalties can be applied multiple times upto a total of 250points" 
As the latest results suggest the task is still pretty hard